src/addresses.h

```diff
@@ -170,8 +156,6 @@
 
 #define RCT2_ADDRESS_CURRENT_SOUND_DEVICE			0x009AF280
 
-#define RCT2_ADDRESS_VEHICLE_SOUND_LIST				0x009AF288
```

Comment:
Keep this one since you will still come across it when implementing vehicle update probably.